pkg/templates/params.go

```diff
@@ -22,18 +22,55 @@ import (
 
 type Params map[string]apiextensionsv1.JSON
 
-func ParamsBuilder(defaultParams v1alpha1.DefaultParams, resourceParams []v1alpha1.Param) Params {
+func ParamsBuilder(
+	templateParams []v1alpha1.TemplateParam,
+	blueprintParams []v1alpha1.DelegatableParam,
```

Comment:
I don't know what blueprint is here. We should discuss terminology. I see that it is for supply chain/delivery from caller

Reply:
Point well taken. I had a callout about the new phrases in PR #354, but didn't copy that part over. I should have!

Rash opened a discussion on the Kubernetes slack about the naming: https://kubernetes.slack.com/archives/C02HKPSEKV1/p1637620952095700

Reply:
deferring for public meeting discussion